serialization.md

```diff
@@ -422,14 +362,13 @@ bin/my-app-binary
 bin/my-app-tools
 ```
 
-The TarSum checksum for the archive file is then computed and placed in the
-JSON metadata along with the execution parameters.
+The TarSum checksum for the archive file is then computed and placed in the JSON metadata along with the execution parameters.
```

Comment:
Please see https://github.com/opencontainers/image-spec/pull/12/files#r59071393. Unfortunately, this specification is out of date.